src/pages/PatientVitals.js

```diff
@@ -138,37 +144,52 @@ const PatientVitals = ({
       ? "page2-sub-wrapper"
       : "page3-sub-wrapper";
 
+  if (!pageLoaded && messageType === MESSAGE_TYPES.newPatient) {
+    return (
+      <div className="loader-wrapper">
+        <div className="loader" />
+      </div>
+    );
+  }
+
   return (
     <div className="wrapper">
       <div className="second-header">
-      {(progressedPage === NEW_PATIENT_PAGES.patientInfo || 
-       progressedPage === NEW_PATIENT_PAGES.covidHistory || 
-       progressedPage ===  NEW_PATIENT_PAGES.preExistingCondition || 
-       progressedPage === NEW_PATIENT_PAGES.allergy || 
-       progressedPage === NEW_PATIENT_PAGES.medication) && 
-        <div className="progress-bar">
-          <div className="progress" ></div>
-          <style>{`
+        {(progressedPage === NEW_PATIENT_PAGES.patientInfo ||
```

Comment:
it looks like it would be easier here to exclude the page that should not this content, then include almost all pages in condition